sdk/dapp-kit/src/components/AccountDropdownMenu.tsx

```diff
@@ -25,42 +32,80 @@ type AccountDropdownMenuProps = {
-            </StyleMarker>
-            <DropdownMenu.Portal>
-                <StyleMarker className={styles.menuContainer}>
-                    <DropdownMenu.Content className={styles.menuContent}>
-                        <div className={styles.scrollableContent}>
-                            {accounts.map((account) => (
-                                <AccountDropdownMenuItem
-                                    key={account.address}
-                                    account={account}
-                                    active={currentAccount.address === account.address}
-                                />
-                            ))}
-                        </div>
-                        <DropdownMenu.Separator className={styles.separator} />
-                        <DropdownMenu.Item
-                            className={clsx(styles.menuItem)}
-                            onSelect={() => disconnectWallet()}
-                        >
-                            Disconnect
-                        </DropdownMenu.Item>
-                    </DropdownMenu.Content>
+        <>
+            <DropdownMenu.Root modal={false}>
+                <StyleMarker>
+                    <DropdownMenu.Trigger asChild>
+                        <Button size={size} className={styles.connectedAccount}>
+                            <Text mono weight="bold">
+                                {currentAccount.label ?? formatAddress(currentAccount.address)}
+                            </Text>
+                            <ChevronIcon />
+                        </Button>
+                    </DropdownMenu.Trigger>
                 </StyleMarker>
-            </DropdownMenu.Portal>
-        </DropdownMenu.Root>
+                <DropdownMenu.Portal>
+                    <StyleMarker className={styles.menuContainer}>
+                        <DropdownMenu.Content className={styles.menuContent}>
+                            <div className={styles.scrollableContent}>
+                                {accounts.map((account) => (
+                                    <AccountDropdownMenuItem
+                                        key={account.address}
+                                        account={account}
+                                        active={currentAccount.address === account.address}
+                                    />
+                                ))}
+                            </div>
+                            {hasIotaConnectMethod && (
+                                <>
+                                    <DropdownMenu.Separator className={styles.separator} />
+                                    <DropdownMenu.Item
+                                        className={clsx(styles.menuItem)}
+                                        onClick={handleManageAccountsClick}
+                                    >
+                                        Manage accounts
+                                    </DropdownMenu.Item>
```

Comment:
i'd change this to `Connect more` instead of `Manage Accounts`, something like:
![image](https://github.com/user-attachments/assets/ba6a6974-a7f9-48d5-9193-6015f05f0227)
wdyt?
```suggestion
                                    <DropdownMenu.Item
                                        className={clsx(
                                            styles.menuItem,
                                            styles.switchAccountMenuItem,
                                        )}
                                        onClick={handleManageAccountsClick}
                                    >
                                        <Text mono>Connect more</Text>
                                        <AddIcon />
                                    </DropdownMenu.Item>
```

Reply:
I think you could also technically disconnect some, so I am not sure about this

Reply:
mmm i'm not sure about `Manage Accounts` either... maybe something like 'Connect/Disconnect Accounts' or something more intuitive :face_with_peeking_eye: